.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,17 @@
+name: lint workflows
+
+on:
+  pull_request:
+    paths:
+      - '.github/**/*.ya?ml'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Add problem matchers
+        run: |
+          # https://github.com/rhysd/actionlint/blob/3a2f2c7/docs/usage.md#problem-matchers
+          echo "::add-matcher::.github/actionlint-matcher.json"
```

Comment:
Not sure about this but it may make sense to [download the matcher](https://github.com/rhysd/actionlint/blob/main/docs/usage.md#problem-matchers:~:text=Copy-,actionlint%2Dmatcher.json,-to%20.github/actionlint), and put it in the cache with the SHA of the file.

Just in case the matches update but doesn't seem to change frequently and it's what the docs recommend, so keeping it like this is fine.

Reply:
I thought about this but I don't think the matcher will be loaded unless we put it in the repo. Let's try.

Reply:
Changed it and it seems to work.

Reply:
Amazing, thanks!